server/server_test.go

```diff
@@ -5296,6 +5296,23 @@ func TestDistributedTracingEnabled(t *testing.T) {
 	}
 }
 
+func TestDistributedTracingResourceAttributes(t *testing.T) {
+	c := []byte(`{"distributed_tracing": {
```

Comment:
You'll need to add `"type": "grpc",` here. Otherwise if you look at the `Init` method, this test currently is not exercising your new code.

Reply:
done